Limitations with mixed PGD text. (#5150)
EnterpriseDB/docs: product_docs/docs/pgd/5/limitations.mdx

Consult the [Durability limitations](durability/limitations) section for a full
and current listing.

## Mixed PGD versions

> PGD has been developed to [enable rolling upgrades of PGD](upgrades) by allowing mixed versions of PGD to run during the upgrade process. 

Author: ```suggestion
PGD has been developed to [enable rolling upgrades of PGD](upgrades) by allowing mixed versions of PGD to operate during the upgrade process. 
```

We expect users to only run mixed versions during upgrades and, once an upgrade has been started, that they complete that upgrade.

> We do not support mixed versions of PGD where there is no upgrade taking place.

Reviewer: Maybe a tad bit crisper:
```suggestion
We do not support running mixed versions of PGD where no upgrade occurs.
```

Author: ```suggestion
We do not support mixed versions of PGD when there is no upgrade taking place.
```

Should really be temporal.

Author: ```suggestion
We do not support running mixed versions of PGD other than while there is an upgrade taking place.
```
